spec/controllers/user_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /users' do
+    # controller => users, action => index
+    subject { get '/users' }
+    before { subject }
+
+    it 'responds successfully with an HTTP 200 status code' do
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders the index template' do
+      expect(response).to render_template(:index)
+    end
+
+    it 'has the correct placeholder text' do
+      expect(response.body).to include('users')
+    end
+  end
+
+  describe 'GET #show' do
+    # controller => users, action => show
+    subject { get '/users/1' }
+    before { subject }
+
+    it 'responds successfully with an HTTP 200 status code' do
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders the show template' do
+      expect(response).to render_template(:show)
+    end
+
+    it 'has the correct placeholder text' do
+      expect(response.body).to include('user')
+    end
```

Comment:
- [ ] Great job do far but I think this tests are NOT passed since after running rspec spec/controllers it shows me this error so kindly let your tests all passed as required ✅✨♻🔥💯💯

![image](https://user-images.githubusercontent.com/58553711/221730041-8d591a13-9c22-4034-b0af-ccbd5424a167.png)

## Here is the error messages to check ⬇⬇
![image](https://user-images.githubusercontent.com/58553711/221730290-0d4a0482-1eaf-41c1-9fb7-409d1b923c3e.png)